### ISSUE TITLE
[Enhancement] Switch to one-page layout for Profile pages

### ISSUE THREAD

**Issue by vpchung** [Sage-Bionetworks/sage-monorepo#1833]
## Sage Design feedback

Current profile pages:

![Image](https://github.com/Sage-Bionetworks/sage-monorepo/assets/9377970/9aac7dc1-ede6-49b8-81f3-c1915bce74ca)

The initial thought was that the sidebar was a _table of contents_ rather than an actual navigation system.  It could be worth to switch to a one-page layout then re-purpose the sidenav as a table of contents?  Especially considering that each of the sub-pages are not going to be long (with the exception of "Stargazers").

**Comment by tschaffter** [comment]
I agree with the idea to show all the content on one page as scrolling is easier than clicking on the different items of the TOC.

That said, I like the TOC and so we could still keep it and instead use it to jump to specific sections of the page. Here the TOC should always be visible and not disappear as the user scroll.

> I was thinking that if we switch to a one-page layout, we could remove "Stargazers" from the table of contents, and instead, link out to the list of users from the metrics div:

I think that the existing "tab" sections should still be kept for potentially long content such as stargazers. On that tab we could have pagination to make it easier to navigate. Future tabs could include detailed analytics such as plot that show data over time, similar challenges / other challenges seen by viewers, etc.

**Comment by rrchai** [comment]
>  It could be worth to switch to a one-page layout then re-purpose the sidenav as a table of contents

I agreeeee~

> Especially considering that each of the sub-pages are not going to be long (with the exception of "Stargazers").

For stargazers, we could create a new page for stargazers list. Then we could set max # of stargazers to display. Next to title of "Stargazers", there is a "see more" or "see all" hyperlink which will redirect to stargazers list page.

**Comment by vpchung** [comment]
> For stargazers, we could create a new page for stargazers list.

Yeah, that's along the same lines I'm thinking.  I like the idea of doing a "See More" button - it follows the same design we're doing on other pages.

**Comment by vpchung** [comment]
### 8/29/2023 update:

I believe this is blocked or on hold, correct, @tschaffter ?

**Comment by tschaffter** [comment]
Let's review this ticket tomorrow in our 1:1 and then decide whether whether we complete/shelve/postpone this ticket.
